C/math/Neon_number.c

```diff
@@ -0,0 +1,37 @@
+// C Program to check whether a number is Neon or not. A neon number is a number where the sum of digits of square of the number is equal to the number.
```

Comment:
Modularize this content: Break it up into multiple lines and use multi-line comments